pages/osx/autopkg.md

```diff
@@ -0,0 +1,35 @@
+# autopkg
+
+> Automation framework for macOS software packaging and distribution.
+
+- Run autopkg recipe:
```

Comment:
We should add the `list-recipes` command first so that users can get the list of recipe identifiers to use in the subsequent commands.